riot/lol/challenges.go

```diff
@@ -0,0 +1,81 @@
+package lol
+
+import (
+	"fmt"
+	log "github.com/sirupsen/logrus"
+	"github.com/yigithanbalci/golio/internal"
+)
+
+type ChallengesClient struct {
+	c *internal.Client
+}
+
+func (cc *ChallengesClient) GetConfig() ([]*ChallengeConfigInfoDto, error) {
+	logger := cc.logger().WithField("method", "GetConfig")
+	var challengeConfigs []*ChallengeConfigInfoDto
+	if err := cc.c.GetInto(endpointChallangesConfig, &challengeConfigs); err != nil {
+		logger.Debug(err)
+		return nil, err
+	}
+	return challengeConfigs, nil
+}
+
+func (cc *ChallengesClient) GetPercentiles() (map[string]map[string]float64, error) {
```

Comment:
Is there no defined structure for the response here? If there is, define a type to use instead of an ad-hoc map.

Reply:
The response type is defined as that map in Riot API docs. Added a custom type for it.